_drafts/blog-post-1.md

```diff
@@ -2,6 +2,8 @@
 layout: post
 title: Student data collection - a helpful history with harmful possibilites 
 author: Emma Raupp
+role: CDL Student Affiliate
+bio: /authors/raupp
 ---
 
 We've all changed to meet the mounting challenges of the 2020 semesters. Faculty and students further adapted to modes of digital learning that have become familiar to us since the outbreak of the novel-coronavirus last Spring, like video-conferencing on Zoom or GoogleMeets and interacting with courses through Canvas. As a student and intern for the CDL, I was especially curious about how students' data was being collected and used during this time, particularly on Canvas. Most of us are aware, to varying degrees, that websites and other platforms we interact with track our data. We may not be fully aware of the scope and means of data tracking and collection, but we make up for this with a suspended sense of wariness surrounding our actions on the Internet. We are advised to proceed with caution in arenas we do not have a clear understanding of, like the overlap between Big Data and the Internet. Being aware of where your data is going (and why) is key to maintaining a sense of autonomy over your digital self. So, what about data collection on an institutional level? This semester, almost every SUNY student submitted assignments, wrote discussion posts, took quizzes, or otherwise interacted with material on Canvas or a similar learning management platform to fulfill their degree requirements. Given our increasing reliance on and trust in Canvas, we should take a moment to explore some important questions: How is student data collected, both on an institutional level and through third party applications like Canvas? What *is* the data being collected? And, perhaps most importantly, to what end is students' data being collected? As students navigating digital learning, it's important that we address these questions and their implications now, both for certainty in our own digital autonomy and for that of future college students.
```

Comment:
- The meaning of "the 2020 semesters" may not be clear to some readers. Maybe *start* with the time frame and then *follow* with what happened in that frame? In other words, something like, "The spring and fall semesters of 2020 presented students and faculty with mounting challenges, and we all changed to meet them." Don't feel you have to adopt that exact wording! It's just an example.
- **As a student and intern for the CDL**: Maybe add one transition sentence ahead of this one? You begin by saying that the challenges of the spring and fall forced people to change and adapt — a great beginning. Maybe you want to then say that these challenges also presented an opportunity to ask some questions? Or that the challenges raised questions? The point is to have some transition that explains why you were "curious" about the use of student data.
- **GoogleMeets**: double check how Google names this product.
- **Spring**: make lower case
- **Internet**: make lower case
- **third party**: hyphenate
- **most importantly**: change to "most important"